types/masonry-layout/index.d.ts

```diff
@@ -71,5 +71,6 @@ declare namespace Masonry {
 declare global {
     interface JQuery {
         masonry(options?: Masonry.Options): JQuery;
+	masonry(selector: string | Element, options?: Masonry.Options | JQuery): JQuery;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/masonry-layout/index.d.ts;
  Line: 74;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.